daemon/k8s/init.go

```diff
@@ -61,7 +62,7 @@ func retrieveNodeInformation(ctx context.Context, log logrus.FieldLogger, localN
 				break
 			}
 			if event.Kind == resource.Upsert {
-				n = k8s.ParseNode(event.Object, source.Unspec)
+				n = k8s.ParseNode(logging.DefaultSlogLogger, event.Object, source.Unspec)
```

Comment:
Is it not possible to pass a logger here, or why are we going with the default?

Reply:
I've all the code migrated to slog already, but to make it easier to review I'm doing PRs on per-package basis. I'm leaving the logging.DefaultSlogLogger on the function callers. At the end the majority of the `logging.DefaultSlogLogger` should be replaced with the root's logger.